spec/models/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.describe User do
+  describe 'with validation' do
```

Comment:
`with` word uses only within `context`
```suggestion
  describe 'validations' do
```